src/index.ts

```diff
@@ -75,7 +77,12 @@ createConnection({
       null,
       true,
     );
-
+    const updateScreenshotsJob = new CronJob('0 */30 * * * *', async () => {
+      if (process.env.chart === 'explorer-chart-worker') {
```

Comment:
NB: better to start move this condition out of cron job, unless env variable isn't going to be changed dynamically